lib/asciidoctor/iso/cleanup.rb

```diff
@@ -86,6 +88,42 @@ def sort_biblio(bib)
         end
       end
 
+      def termdef_cleanup(xmldoc)
+        generate_termref_links(xmldoc)
+        super
+      end
+
+      def generate_termref_links(xmldoc)
+        termlookup = replace_automatic_genrated_ids_terms(xmldoc)
+        xmldoc.xpath('//termxref').each do |node|
+          target = node.text
+          next if termlookup[target].nil?
```

Comment:
Please log an error here, using `@log.add("AsciiDoc Input", node, "#{target} does not refer to a real term")`

Reply:
Done